src/platform/monitoring/DowntimeNotification/components/DowntimeApproaching.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Modal from '@department-of-veterans-affairs/formation/Modal';
+import serviceStatus from '../config/serviceStatus';
+import dismissedDowntimeNotifications from '../util/dismissedDowntimeNotifications';
+import DowntimeNotificationWrapper from './Wrapper';
+
+export default class DowntimeApproaching extends React.Component {
+  constructor(props) {
+    super(props);
+    dismissedDowntimeNotifications.setup();
+    this.state = {
+      modalDismissed: dismissedDowntimeNotifications.contains(props.appTitle),
+      cache: {}
+    };
+  }
+
+  dismissModal = () => {
+    dismissedDowntimeNotifications.push(this.props.appTitle);
+    this.setState({ modalDismissed: true });
```

Comment:
Do we need to store this as a state property? Or could we create a modalDismissed variable in render instead? Perhaps that would clutter render.

Reply:
I actually just realized that instead of storing a local variable in render we can just pass the `this.state.modalDismissed` property to the Modal component. That should clean things up some.

I'm not sure how a modalDismissed variable not stored in state though could be used though. Maybe by moving it into the parent scope?

Reply:
Good point, that's a better idea.

Reply:
Another option is to store the list in Redux, which is what you would do in a stricter Redux app (again, would be good to have some agreed upon guidelines)